api/src/main/java/com/velocitypowered/api/event/proxy/ServerRegisteredEvent.java

```diff
@@ -0,0 +1,25 @@
+package com.velocitypowered.api.event.proxy;
+
+import com.google.common.base.Preconditions;
+import com.velocitypowered.api.proxy.server.ServerInfo;
+
+/**
+ * This event is fired by the proxy after a new backend server is registered
```

Comment:
This should probably explain a bit more of what "registered" means

Reply:
"registered" means adding a new backend server to the proxy dynamically at runtime.

Will this do?

Reply:
Will do